components/o-message/stories/message.stories.tsx

```diff
@@ -0,0 +1,192 @@
+import withHtml from 'origami-storybook-addon-html';
+import {withDesign} from 'storybook-addon-designs';
+import {ComponentStory, ComponentMeta} from '@storybook/react'
+import { useEffect } from 'react';
+import {Message} from '../src/tsx/message';
+import javascript from '../main';
+import './message.scss'
+
+
+export default {
+	title: 'Components/o-message',
+	component: Message,
+	decorators: [withDesign, withHtml],
+	// These params are copied from banner.tsx. need to find corresponding design in figma
+	parameters: {
+		design: {
+			type: 'figma',
+			url: 'https://www.figma.com/file/MyHQ1qdwYyek5IBdhEEaND/FT-UI-Library?node-id=0%3A1489',
+		},
+		guidelines: {
+			notion: '37cdc7ac2cac4d60a4c9f451c47a4647',
+		},
+		html: {},
+	},
+	args: {},
+  } as ComponentMeta<typeof Message>;
+
+const Story = args => {
+	useEffect(() => {
+		let messages = javascript.init(null, null)
+		return function cleanup() {
+			messages = Array.isArray(messages) ? messages : [messages];
+			messages.forEach(message => message.destroy());
+		};
+	}, [args.showCloseButton])
```

Comment:
I copied this code from `Banner.tsx` and it works. I get the idea but `.destroy()` seems to be unnecessary. since message does not have function that destroys it. do we know what happens if we don't use `useEffect()` and just initialise javascript?

Reply:
```suggestion
	useEffect(() => void javascript.init(), [])
```

this is probably fine